sips/sip-168.md

```diff
@@ -0,0 +1,129 @@
+---
+sip: 168
+title: Governance Participation Program
+author: Andy T CF (@andytcf), MiLLiΞ (@Milli3E)
+discussions-to: https://research.synthetix.io/t/governance-participation-program/450
+status: Draft
+created: 2021-07-19
+---
+
+## Implementor
+
+Andy T CF (@andytcf)
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the SCCP.-->
+
+This proposal aims to encourage governance participation from the Synthetix community and ecosystem abroad. Token holders that have values aligned with that of the Ambassadors will now be rewarded with SNX for delegating their votes for the duration of an Epoch (initially set at 28 days), to the Ambassador Multi-sig.
```

Comment:
@andytcf how much SNX will they be rewarded with, or if not sure can you add commentary on how that will be decided? Such as determined by the sDAO etc?